travis-scripts/valgrind.sh

```diff
@@ -162,21 +171,24 @@ echo "Running update.cf:"
 valgrind $VG_OPTS /var/cfengine/bin/cf-agent -K -f update.cf 2>&1 | tee update.txt
 check_output update.txt
 check_masterfiles_and_inputs
+
 echo "Running update.cf without local copy:"
 valgrind $VG_OPTS /var/cfengine/bin/cf-agent -K -f update.cf -D mpf_skip_local_copy_optimization 2>&1 | tee update2.txt
 check_output update2.txt
 check_masterfiles_and_inputs
+
 echo "Running promises.cf:"
 # The mpf limits concurrency of daemons (namely cf-execd and cf-monitord) using a loose process check
-valgrind $VG_OPTS /var/cfengine/bin/cf-agent -K -f promises.cf 2>&1 | tee promises.txt
+valgrind $VG_OPTS /var/cfengine/bin/cf-agent -K --define mpf_disable_cfe_internal_limit_robot_agents -f promises.cf 2>&1 | tee promises.txt
```

Comment:
I don't think this is necessary, since my PR is green without it:

https://github.com/cfengine/masterfiles/pull/1539